Project.toml

```diff
@@ -1,12 +1,13 @@
 name = "RDatasets"
 uuid = "ce6b1742-4840-55fa-b093-852dadbb1d8b"
-version = "0.7.7"
+version = "0.7.8"
```

Comment:
since you change functionality probably 0.8 is a better version number.

Reply:
```suggestion
version = "0.8.0"